src/components/projects/create-wp-form/create-wp-form.tsx

```diff
@@ -3,23 +3,46 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { useState } from "react";
-import { useHistory } from "react-router-dom";
-import { isProject, validateWBS, WbsNumber } from "utils";
-import CreateWPFormView from "./create-wp-form/create-wp-form";
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { isProject, validateWBS, WbsNumber } from 'utils';
+import { useAuth } from '../../../services/auth.hooks';
+import { useCreateSingleWorkPackage } from '../../../services/work-packages.hooks';
+import { routes } from '../../../shared/routes';
+import CreateWPFormView from './create-wp-form/create-wp-form';
 
 export interface EditableTextInputListUtils {
   add: (val: any) => void;
   remove: (idx: number) => void;
   update: (idx: number, val: any) => void;
 }
 
+export interface StateBundle {
+  state: any;
+  setter: (val: any) => void;
```

Comment:
Pretty sure there's a better definition for this type

Reply:
https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/src/components/change-requests/review-change-request/review-change-request/review-change-request.tsx#L15

Reply:
Also this interface belongs elsewhere probably. Maybe in the shared folder in a new file? Or something along the lines of component supports tools?

Reply:
Yeah, I realized after typing that out that I didn't actually need the state at all, so I'm going to remove that interface and just use the type from your referenced example.